### PR TITLE
Medical AI - Add setting to require items for treatment

### PR DESCRIPTION
**Pull request by PabstMirror**

### CODE REVIEW

**Comment by PabstMirror** [comment]
could do a one-time call to `_healer call ace_common_fnc_replaceRegisteredItems`
but I don't think it's needed,
this setting is for people who are probably going to be setting custom AI loadouts

File: addons/medical_ai/stringtable.xml [hunk 1]
```diff
@@ -48,5 +48,11 @@
             <Czech>Pouze Server a HC</Czech>
             <Turkish>Sadece Sunucu ve HC de</Turkish>
         </Key>
+        <Key ID="STR_ACE_Medical_AI_requireItems_title">
+            <English>Require Items</English>
+        </Key>
+        <Key ID="STR_ACE_Medical_AI_requireItems_desc">
+            <English>AI healers will only treat if they have the necessary ACE medical items in their loadout</English>
```
Comment (LinkIsGrim) [inline addons/medical_ai/stringtable.xml:55]:
```suggestion
            <English>AI will only perform medical treatmet if they have the necessary items in their inventory.\nRequires custom AI loadouts!</English>
```

**Review by LinkIsGrim — APPROVED** [review]

**Review by LinkIsGrim — APPROVED** [review]

**Comment by acknowledge14** [comment]
HI, I wanna ask, if this change applied, will ace_medical_treatment_convertitems also happend on AI soldiers?
eg. vanilla FAK converts to ACE bandages and morphine

**Comment by LinkIsGrim** [comment]
Not on AI, no.

**Comment by PabstMirror** [comment]
```
["CAManBase", "init", {
    params ["_unit"];
    if (local _unit) then {
        [_unit] call ace_common_fnc_replaceRegisteredItems
    };
}] call CBA_fnc_addClassEventHandler;
```

**Pull request merged**